nucleus/dataset.py

```diff
@@ -168,9 +167,6 @@ def annotate(
             "ignored_items": int,
         }
         """
-        if any((isinstance(ann, CuboidAnnotation) for ann in annotations)):
```

Comment:
still need this check in dataset.annotate since you can't upload them yet

Reply:
cuboids can be uploaded, the backend changes are already there. here's the tests for that:
[tests](https://github.com/scaleapi/scaleapi/blob/master/test/nucleus/testEndpoints.ts#L786)

Reply:
should add test coverage for this here then to make sure the format generated by the client is compatible with the backend